
(node:1070733) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_2',
  containerIp: '10.0.3.48',
  mitmPort: 1001,
  autoAccessFixed: 1
}
2022-10-25 19:44:02.120 - [Info] MITM Version: 2
2022-10-25 19:44:02.121 - [Info] Auto Access Enabled: true
2022-10-25 19:44:02.122 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 19:44:02.129 - [Info] SSH man-in-the-middle server for 10.0.3.48 listening on 10.0.3.1:1001
(node:1082609) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
2022-10-25 19:45:13.726 - [Debug] [Connection] Attacker connected: 5.28.139.161 | Client Identification: SSH-2.0-Go
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_2',
  containerIp: '10.0.3.180',
  mitmPort: 1001,
  autoAccessFixed: 1
}
2022-10-25 19:45:13.777 - [Info] MITM Version: 2
2022-10-25 19:45:13.778 - [Info] Auto Access Enabled: true
2022-10-25 19:45:13.779 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 19:45:13.787 - [Debug] listen EADDRINUSE: address already in use 10.0.3.1:1001
2022-10-25 19:45:13.787 - [Error] Another MITM instance or another program is already listening on this port
{"level":"error","message":"Forever detected script exited with code: 1"}
2022-10-25 19:45:13.898 - [Debug] [Connection] Attacker connected: 69.236.215.6 | Client Identification: SSH-2.0-Go
2022-10-25 19:45:14.141 - [Debug] [Auth] Attacker 69.236.215.6 trying to authenticate with 'none'
2022-10-25 19:45:14.142 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 19:45:14.241 - [Debug] [Auth] Attacker 69.236.215.6 trying to authenticate with 'publickey'
2022-10-25 19:45:14.242 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 19:45:14.243 - [Debug] [Auth] Attacker: 69.236.215.6 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 19:45:14.357 - [Debug] [Connection] Attacker closed the connection
2022-10-25 19:45:14.389 - [Debug] [Connection] Attacker connected: 69.236.215.6 | Client Identification: SSH-2.0-Go
